scripts/appscale_java8_apiserver.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Launches the AppScale Java 8 API server
+
+API_ARGS=""
+while test -n "${1}"; do
+  case "${1}" in
+    --application_prefix)
+      shift
+      ;;
+    --clear_datastore)
+      ;;
```

Comment:
I assume the missing shift is on purpose: is this the only extra option we would like to pass on?

Reply:
The matched arguments are dropped, when there is a related value it is also dropped. For `--clear_datastore` it is just a flag, so there is nothing else to drop.